product_main_supplier/README.rst

```diff
@@ -0,0 +1,58 @@
+=====================
+Product Main Supplier
+=====================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-coopiteasy%2Fobeesdoo-lightgray.png?logo=github
+    :target: https://github.com/coopiteasy/obeesdoo/tree/12.0/product_main_supplier
+    :alt: coopiteasy/obeesdoo
+
+|badge1| |badge2| |badge3| 
+
+Add a main supplier
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/coopiteasy/obeesdoo/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed
+`feedback <https://github.com/coopiteasy/obeesdoo/issues/new?body=module:%20product_main_supplier%0Aversion:%2012.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+Do not contact contributors directly about support or help with technical issues.
+
+Credits
+=======
+
+Authors
+~~~~~~~
+
+* Coop IT Easy SCRLfs
+
+Contributors
+~~~~~~~~~~~~
+
+* Beescoop - Cellule IT
```

Comment:
This readme should be regenerated again (because Contributors does not reflect the `CONTRIBUTOR` file.